mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// Math.random() * 10 returns a random number between 0 (included) and 10 (excluded);
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+// The concat() method joins two or more strings.
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
This is good 👍 
Can you think of any other ways to do this?
Maybe https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals